mllib/src/test/scala/org/apache/spark/ml/clustering/PowerIterationClusteringSuite.scala

```diff
@@ -76,23 +78,31 @@ class PowerIterationClusteringSuite extends SparkFunSuite
       .setMaxIter(40)
       .setWeightCol("weight")
       .assignClusters(data)
-    val localAssignments = assignments
-      .select('id, 'cluster)
-      .as[(Long, Int)].collect().toSet
-    val expectedResult = (0 until n1).map(x => (x, 1)).toSet ++
-      (n1 until n).map(x => (x, 0)).toSet
-    assert(localAssignments === expectedResult)
+      .select("id", "cluster")
+      .as[(Long, Int)]
+      .collect()
+
+    val predictions = Array.fill(2)(mutable.Set.empty[Long])
+    assignments.foreach{
```

Comment:
This might fail the style checker -- need a space before the brace. Looks good otherwise. Let's see if the test passes.

Reply:
I have added the space. Thank you.